friendships/tests.py

```diff
@@ -106,3 +109,28 @@ def test_friendship_str(self):
                   f'{self.friendship_1_3.to_user}-{self.friendship_1_3.to_user_id} ' + \
                   f'at {self.friendship_1_3.created_at} --'
         self.assertEqual(message, str(self.friendship_1_3))
+
+
+class FriendshipServiceTest(TestCase):
+
+    def setUp(self):
+        self.clear_cache()
+        self.kd35 = self.create_user(username='nets_kd35')
+        self.mj23 = self.create_user(username='bulls_mj23')
+
+    def test_get_followings(self):
+        kd35_following_id_set = set()
+        for i in range(3):
+            user = self.create_user(username=f'kd35_following:{i}')
+            kd35_following_id_set.add(user.id)
+            Friendship.objects.create(from_user=self.kd35, to_user=user)
+        Friendship.objects.create(from_user=self.kd35, to_user=self.mj23)
+        kd35_following_id_set.add(self.mj23.id)
+
+        following_user_id_set = FriendshipService.get_following_user_id_set(self.kd35.id)
+        self.assertSetEqual(following_user_id_set, kd35_following_id_set)
```

Comment:
should call get_following_user_id_set() twice to test cache.
Get followings from DB and set to cache first time, and get followings from cache second time

Reply:
Resolved as suggestion